Kwenta/kwenta
# Fix: Socket re-rendering issue upon network switch

## Description
When the user is connected to the non-op network, the trade panel will be hidden, so the socket widget will be re-rendered. The PR is fixing this issue. Switching the network won't re-render the socket widget when the `Bridge & Swap` modal is open.

## Related issue
Closes #2576 

## Motivation and Context


## How Has This Been Tested?
Case: User is connected to the Optimism Mainnet without any sUSD
1. Click on `Get sUSD`
2. Set the `Source` to `Ethereum` and `Token` to `Ether`
3. Set the `Destination` to `Optimism` and `Token` to `sUSD`
4. Click the `Review Quote` and `Confirm Bridge`
5. Click the `Switch chain to Ethereum`
6. Click the `Bridge`

## Screenshots (if appropriate):
Switching the network won't re-render the socket widget when the `Bridge & Swap` modal is open.
<img width="1840" alt="截屏2023-07-18 21 55 52" src="https://github.com/Kwenta/kwenta/assets/4819006/f6887cbc-a8c4-473d-9eb5-77e401c5de79">
